src/site/content/en/react/code-splitting-suspense/index.md

```diff
@@ -95,11 +95,11 @@ weak network connections.
 
 To better demonstrate how this works:
 
-+  Mouse over the editor and press the **Show** button to preview the app.
-+  Open the DevTools by pressing `CMD + OPTION + i` / `CTRL + SHIFT + i`.
-+  Click on the **Network** tab.
-+  Click the [**Throttling**](https://developers.google.com/web/tools/chrome-devtools/network/#throttle) dropdown and select `Slow 3G`.
-+  Click the button on the application.
+1.  Mouse over the editor and press the **Show** button to preview the app.
+1.  Open the DevTools by pressing `CMD + OPTION + i` / `CTRL + SHIFT + i`.
+1.  Click on the **Network** tab.
+1.  Click the [**Throttling**](https://developers.google.com/web/tools/chrome-devtools/network/#throttle) dropdown and select `Slow 3G`.
+1.  Click the button on the application.
```

Comment:
I think we have a [component](https://web.dev/handbook/web-dev-components/) that we should be using here instead, but we can fix that in a follow-up PR.